app/goal_routes.py

```diff
@@ -0,0 +1,99 @@
+
+from datetime import datetime
+from flask import Blueprint, jsonify, make_response, request, abort
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from sqlalchemy import desc #(used in read_all_tasks for sorting)
+import os 
+from .task_routes import validate_model
+
+goals_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+#CREATE Routes (Wave 5: CRUD Routes for goal model)
+
+@goals_bp.route('', methods=['POST'])
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except KeyError:
+        return jsonify ({"details": "Invalid data"}), 400
```

Comment:
Nice work with try-except, but it would be good to provide the user information about what data is invalid.